packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,86 @@
+import os
+import time
+
+import requests
+
+from test_packaging import (
+    _skipif_insufficient_resources
+)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
+
+
+def _get_tweet_count(address):
+    """Count the number of tweets on the tweeter app's webpage
+
+    Args:
+        address: str URL including http://
+    """
+    response = requests.get(address)
+    return response.text.count('tweet-content')
+
+
+def test_tweeter_demo_oss(dcos_api_session):
+    """Step through setup and run the Tweeter demo application. See https://github.com/mesosphere/tweeter
+    Should be equivalent to the cli script. It passes if service and app installations are successful and the
+    post_tweets application successfully posts tweets to the tweeter webapp.
+
+    Service versions are read through environment variables set through the Tweeter Demo jov on TeamCity. These default
+    to None, so cosmos/packaging will install the most recent version of the service compatible with the version of
+    DC/OS being used.
+
+    Note that the Zeppelin notebook is not run, nor is it set up with cores_max parameter to 8 as noted in the docs for
+    manually running this.
+    """
+    _skipif_insufficient_resources(dcos_api_session, DEMO_RESOURCE_REQUIREMENTS)
+
+    dependencies = {
+        'cassandra': os.environ.get('CASSANDRA_VERSION'),
+        'kafka': os.environ.get('KAFKA_VERSION'),
+        'zeppelin': os.environ.get('ZEPPELIN_VERSION'),
+        'marathon-lb': os.environ.get('MARATHON_LB_VERSION')
+    }
+
+    for service, version in dependencies.items():
+        print("Installing {0} {1}".format(service, version or "(most recent version)"))
```

Comment:
Please reserve using `print` for output information that wouldnt be appropriate in a log stream, such as dumping a file or bulk test metrics